miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/KeyboardBridgeDispatcher.kt

```diff
@@ -0,0 +1,31 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import android.app.Activity
+import android.content.Context
+import android.view.inputmethod.InputMethodManager
+
+internal class KeyboardBridgeDispatcher(
+    private val bridgeExecutor: MiniAppBridgeExecutor,
+    private val activity: Activity
+) {
+
+    fun onGetKeyboardVisibility(callbackId: String) {
+        try {
+            val inputMethodManager: InputMethodManager =
+                activity.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+
+            if (inputMethodManager.isActive && inputMethodManager.isAcceptingText)
```

Comment:
There is a case, when an input field inside JS MiniApp has a focus but keyboard is hidden, then `inputMethodManager.isAcceptingText` will return `true` means Keyboard is shown. So, this solution doesn't solve this use-case.

Reply:
Solved by https://github.com/rakutentech/android-miniapp/pull/174/commits/15df9f7f046621ecc91ccb40f8a2c772aa702946